File: Services/ToDoTask/ToDoTask.API/Entities/TaskItem.cs
using System;

namespace ToDoTask.API.Entities
{
    public class TaskItem
    {
        public int Id { get; set; }
        public string TaskName { get; set; }
        public string Description { get; set; }
        public DateTime? DueDate { get; set; }
        public string Assignee { get; set; }
        public string Status { get; set; }
        public string[] Label { get; set; }
        public string Priority { get; set; }
    }
}

File: client/src/app/models/TaskItem.ts
export interface TaskItem {
    id?: number;
    taskName: string;
    description?: string;
    dueDate?: Date;
    assignee: string;
    status: string;
    label?: string[];
    priority?: string;
}

export interface TaskItemParams {
    orderBy: string;
    searchTerm?: string;
    dueDate?: string;
    priorities: string[];
    labels: string[];
    status?: string;
}